update approver/reviewer for jp

nasa9084 opened the pull request.

nasa9084 committed:
update approver/reviewer for jp

nasa9084 commented:
/assign @zacharysarah

xiangpengzhao commented on OWNERS:
```diff
@@ -1,22 +1,13 @@
-- zparnold
+- cstoku
+- nasa9084
+- tnir
```
Are these reviewers and approvers all members of Kubernetes organization? If not, I'm afraid they may not be able to lgtm/approve PRs in this repo.

xiangpengzhao commented:
/assign @tnir
/assign @cstoku 
for testing

zacharysarah commented:
@nasa9084 Thanks for updating these! :sparkles:

1. The `OWNERS` file looks good. 

@cstoku: Would you like to keep myself and @chenopis (as SIG Leads) as approvers? It isn't necessary, but it may be useful sometimes. If this PR is approved as is, I will no longer be able to approve any following PRs.

2. Please add [the following block](https://github.com/kubernetes/kubernetes-docs-zh/blob/master/OWNERS_ALIASES) back to `OWNERS_ALIASES`:

```
aliases:
  # Emptied for localization
  # SIGs may add specific reviewers with fluency in the localization
```

3. Thanks for providing a list of `SECURITY_CONTACTS`. :+1: Please keep myself and @chenopis as security contacts; as leads for SIG Docs, we need to know about any security concerns with a localization repo.

nasa9084 committed:
add empty aliases block

@zacharysarah said:

> Please add the following block back to OWNERS_ALIASES:
>
> aliases:
>   # Emptied for localization
>   # SIGs may add specific reviewers with fluency in the localization

nasa9084 committed:
keep @zacharysarah and @chenopis as approvers

@zacharysarah said:

> @cstoku: Would you like to keep myself and @chenopis (as SIG Leads) as approvers? It isn't necessary, but it may be useful sometimes. If this PR is approved as is, I will no longer be able to approve any following PRs.

zacharysarah commented:
Thanks for the test, @xiangpengzhao. 🙇 

@tnir 👋You need to join the Kubernetes org as a [member](https://github.com/kubernetes/community/blob/master/community-membership.md#member) before you can be a reviewer/approver.

nasa9084 committed:
keep @zacharysarah and @chenopis as SECURITY CONTACTS

@zacharysarah said:

> Thanks for providing a list of SECURITY_CONTACTS. 👍 Please keep myself and @chenopis as security contacts; as leads for SIG Docs, we need to know about any security concerns with a localization repo.

xiangpengzhao commented:
:smile: 

/cc @makocchi-git @MasayaAoyama
/assign @nasa9084

This is a test for the other three folks. Those who are not org members need to join :)

nasa9084 commented:
I created 3 commits for @zacharysarah comment.

* keep @zacharysarah  and @chenopis (as SIG Leads) as approvers
* add the block to OWNERS_ALIASES
* Please keep @zacharysarah  and @chenopis as security contacts

cstoku commented:
I agree to keep @zacharysarah and @chenopis as approvers!

Everyone except me isn't Kubernetes org member.
But these people are doing great activities at Kubernetes community.
Would it be possible for @zacharysarah and @xiangpengzhao to become their sponsor?
Or, How about becoming an approver just me at another PR first and becoming one of the their sponsor?
(if so, will @zacharysarah be another sponsor of them?)

xiangpengzhao commented:
> Would it be possible for @zacharysarah and @xiangpengzhao to become their sponsor?

Sure :)

In case someone don't know the request process, below is the link:
https://github.com/kubernetes/community/blob/master/community-membership.md#member

makocchi-ca commented:
@xiangpengzhao Thank you for your kindness!

makocchi-git commented:
@xiangpengzhao Thank you for your kindness!

MasayaAoyama commented:
@xiangpengzhao @zacharysarah @cstoku 
Thank you for your help!

zacharysarah commented:
@nasa9084 Let's hold this PR until @makocchi-git and @MasayaAoyama are approved as members. When they are members, they can `/hold cancel`. 😄 🎉 

/hold

nasa9084 commented:
It's OK. I also has been requesting membership.

tnir commented:
/lgtm

zacharysarah commented:
/approve

cstoku commented:
@zacharysarah @xiangpengzhao  Thank you very much for your help!!
